.gitignore

```diff
@@ -2,3 +2,16 @@ node_modules
 .idea
 bin/
 npm-debug.log
+
+# OS generated files #
```

Comment:
no worries @Hacker-YHJ really appreciate the PR here and your thoughts :)

Feel free to open up any other PRs or issues!